cumulusci/tasks/bulkdata/load.py

```diff
@@ -451,7 +454,12 @@ def _init_db(self):
                 self.models = {}
                 for name, mapping in self.mapping.items():
                     if mapping.table not in self.models:
-                        self.models[mapping.table] = self.base.classes[mapping.table]
+                        try:
+                            self.models[mapping.table] = self.base.classes[
+                                mapping.table
+                            ]
+                        except KeyError as e:
+                            raise BulkDataException(f"Table not found in dataset: {e}")
```

Comment:
Thanks for fixing this while you're in here -- I can cross that off my list. Please mention it under Issues closed in this PR's description.